tests/unit/mathematics/linalg/LinalgBackendViennaCL_unittest.cc

```diff
@@ -88,13 +88,17 @@ TEST(LinalgBackendViennaCL, SGVector_clone)
 	sg_linalg->set_gpu_backend(new LinalgBackendViennaCL());
 
 	const index_t size = 10;
-	SGVector<int32_t> a(size), b(size), c(size);
+	SGVector<int32_t> a(size);
 	a.range_fill(0);
```

Comment:
@karlnapf Sorry I was wrong about it. I thought the `operator=` was not overloaded but when I was writing the comment I realized it was defined in the base class. So the old way was fine but I prefer the new one, as it avoids the unnecessary memory allocating and releasing.